testdata/main.go

```diff
@@ -136,4 +137,6 @@ func main() {
 	b2.Write(nil)
 	rand.Read(nil)
 	mrand.Read(nil)
+
+	ioutil.ReadFile("main.go") // UNCHECKED
```

Comment:
But basically the idea is to have an exclude file with `ioutil.ReadFile` in it and then this line should not be flagged as an unchecked error. I just need to figure out a good way to tell the test case that.